[2021-10-14 14:14:36,674] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: example_trigger_controller_dag.test_trigger_dagrun scheduled__2021-01-01T00:00:00+00:00 [queued]>
[2021-10-14 14:14:36,722] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: example_trigger_controller_dag.test_trigger_dagrun scheduled__2021-01-01T00:00:00+00:00 [queued]>
[2021-10-14 14:14:36,723] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-10-14 14:14:36,723] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-10-14 14:14:36,724] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-10-14 14:14:36,747] {taskinstance.py:1262} INFO - Executing <Task(TriggerDagRunOperator): test_trigger_dagrun> on 2021-01-01 00:00:00+00:00
[2021-10-14 14:14:36,760] {standard_task_runner.py:52} INFO - Started process 942 to run task
[2021-10-14 14:14:36,774] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'example_trigger_controller_dag', 'test_trigger_dagrun', 'scheduled__2021-01-01T00:00:00+00:00', '--job-id', '34', '--raw', '--subdir', '/home/***/.local/lib/python3.6/site-packages/***/example_dags/example_trigger_controller_dag.py', '--cfg-path', '/tmp/tmpj_jd8y4o', '--error-file', '/tmp/tmplkfpukna']
[2021-10-14 14:14:36,776] {standard_task_runner.py:77} INFO - Job 34: Subtask test_trigger_dagrun
[2021-10-14 14:14:36,850] {logging_mixin.py:109} INFO - Running <TaskInstance: example_trigger_controller_dag.test_trigger_dagrun scheduled__2021-01-01T00:00:00+00:00 [running]> on host 522ccbfd6863
[2021-10-14 14:14:36,925] {taskinstance.py:1414} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=example_trigger_controller_dag
AIRFLOW_CTX_TASK_ID=test_trigger_dagrun
AIRFLOW_CTX_EXECUTION_DATE=2021-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-01-01T00:00:00+00:00
[2021-10-14 14:14:36,974] {warnings.py:99} WARNING - /home/***/.local/lib/python3.6/site-packages/***/api/common/experimental/trigger_dag.py:89: DeprecationWarning: Calling `DAG.create_dagrun()` without an explicit data interval is deprecated
  dag_hash=dag_bag.dags_hash.get(dag_id),

[2021-10-14 14:14:37,018] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=example_trigger_controller_dag, task_id=test_trigger_dagrun, execution_date=20210101T000000, start_date=20211014T141436, end_date=20211014T141437
[2021-10-14 14:14:37,052] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-10-14 14:14:37,088] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
